src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_b_fs_yx_fsv_16_32_imad_dw.cpp

```diff
@@ -83,9 +84,16 @@ bool ConvolutionKernel_b_fs_yx_fsv_16_32_imad_dw::Validate(const Params& params)
     if (conv_params.inputs[0].GetLayout() != conv_params.outputs[0].GetLayout())
         return false;
 
+    if (conv_params.inputs[0].Feature().is_dynamic || conv_params.outputs[0].Feature().is_dynamic)
+        return false;
+
     if (conv_params.groups != conv_params.outputs[0].Feature().v || conv_params.groups != conv_params.inputs[0].Feature().v)
         return false;
 
+    // Incorrect choose of TILE_X leads to accuracy issue
+    if (conv_params.outputs[0].X().is_dynamic)
+        return false;
```

Comment:
Doesn't it make sense to allow dynamic X dimension with some default TILE_X=1 value? Wouldn't it perform better than ref kernel?

Reply:
For some reason using TILE_X=1 by default causes accuracy issues
This can be debugged and fixed later as another step to improve performance